### ISSUE TITLE
Mouse issue.. ghost in the machine.

### ISSUE THREAD

**Issue by BTBlueSkies** [pop-os/pop#1046]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
Mouse looses events randomly... 

I click on occasion and the mouse event is 'lost' to the system. Sometimes I am slow and click and then click again.. I catch the issue every now and then but I cannot replicate it in any meaningful way. However, that's not to say this is not a real issue. This is a multi boot machine and the hardware is fine.. so it has to be in the pop-ubuntu-system

In short, randomly mouse clicks don't register, not always, but sometimes. A click here or there is lost. I think others may be seeing the same thing but it's a guess. I'd say at least a few times an hour when working actively. 

It's very noticeable when I click into a window and notice it doesn't highlight, then I click again and yet a third time.

The problem is sometime I click to activate and 'knowing I clicked the active window under the mouse' and start to type... well the typing goes into a some other window I am not sure where it when but... after several times hitting the build button on a project and finding those characters.. it's a problem.

I am posting this hoping others may notice the same thing and can chime in and confirm.

**Steps to reproduce (if you know):**

**Expected behavior:**

**Other Notes:**

**Comment by Oymate** [comment]
@BTBlueSkies Does the mouse cursor become slow?

**Comment by BTBlueSkies** [comment]
No, It behaves normally.  The mouse is quick and responsive.. and the (side note) jitter that I was noticing in 19.10 was resolved with 20.04

**Comment by Oymate** [comment]
@BTBlueSkies I was also facing something similar after the update. But, following a article resolved it.
https://github.com/pop-os/default-settings/issues/85
